packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -642,7 +652,7 @@ export abstract class SharedObjectCore<
 	 * @param args - Arguments to pass to the event listeners.
 	 * @returns `true` if the event had listeners, `false` otherwise.
 	 */
-	public emit(event: EventEmitterEventType, ...args: any[]): boolean {
+	public emit(event: EventEmitterEventType, ...args: unknown[]): boolean {
```

Comment:
See my note in the API report - rest parameters should be okay to type as `any[]`, and probably should be typed that way.